addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3765,7 +3765,17 @@ msgctxt "#837"
 msgid "Sundays"
 msgstr ""
 
-#empty strings from id 838 to 839
+#. Label for a TV show Premiere
+#: addons/skin.estuary/xml/DialogPVRInfo.xml
+msgctxt "#838"
+msgid "Première"
```

Comment:
Think here you have a typo, sounds a little French with `è` :smirk: .

Reply:
Nope, not a typo. You can spell with or without the accent in English!

Reply:
Learned something again, thanks. I did not know yet.